tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "module": "CommonJS",
+    "module": "node16",
```

Comment:
Are we in node18? Or not yet?

Reply:
It refers to the module resolution, not really the node version https://www.typescriptlang.org/tsconfig#node16nodenext-nightly-builds